apps/docs/src/app/platform/component-docs/platform-forms/checkbox/platform-checkbox-examples/platform-multiselect-checkbox.component.html

```diff
@@ -0,0 +1,167 @@
+<h3>Template Driven Form</h3>
+<p>Checkbox can be pre-selected if value of checkbox matches any value in ngModel directive.</p>
+<fdp-form-group [multiLayout]="true">
+    <fdp-form-field [id]="'sirius'" zone="zLeft" rank="1">
+        <fdp-checkbox [name]="'sirius'" [label]="'Sirius'" [checkboxValue]="'sirius'" [(ngModel)]="field1"></fdp-checkbox>
+    </fdp-form-field>
+
+    <fdp-form-field [id]="'vega'" zone="zRight" rank="1">
+        <fdp-checkbox [name]="'vega'" [label]="'Vega'" [checkboxValue]="'vega'" [(ngModel)]="field2"></fdp-checkbox>
+    </fdp-form-field>
+</fdp-form-group>
+<div style="padding-left: 2rem;">field1: {{ field1 }}</div>
+<div style="padding-left: 2rem;">field2: {{ field2 }}</div>
+<br />
+<br />
+
+<p>Multiple checkboxes connected with same ngModel directive. Multiple checkbox can be selected.</p>
```

Comment:
this example is a bit miss-leading. Usually when try have CheckboxGroup, those checkboxes should have identical name. and most likely we will have this covered with CheckboxGroup. This will be platform Solution. To group multiple checkboxes we should have reference here to different component.

Reply:
reference here to different component :-- this i did not understand. please elaborate.

Reply:
have removed multiple checkbox example. will be covered in Checkbox group.
Have mentioned this in documentation.